### PR TITLE
EPMRPP-81518 || Update components colors and opacity for icons, buttons

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss [hunk 1]
```diff
@@ -38,6 +38,19 @@
   justify-content: center;
   width: 100%;
   margin-top: 5px;
+
+  svg path {
+    fill: $COLOR--e-300;
+  }
+
+  /* stylelint-disable */
+  [class*="with-popover"] {
```
Comment (Bam6ycha) [inline app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss:47]: I am not quite sure, but looks like we can solve this using another approach

app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

Find IconShow component and wrap Parser(icon) with span f.e.
And you can change necessary properties without important and such complex selectors

Comment (BlazarQSO) [reply app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss]: Done.

**Review by Bam6ycha — APPROVED** [review]

File: app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss [hunk 2]
```diff
@@ -79,4 +92,20 @@
...
+        color: $COLOR--topaz-hover-2;
+      }
+      &:focus {
+        color: $COLOR--topaz-focused;
+      }
+      &:active {
+        color: $COLOR--topaz-pressed;
+      }
+    }
+  }
```
Comment (AmsterGet) [inline app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss:110]: Why are these changes not done in the `Button` component?

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by Vadim73i — APPROVED** [review]

File: app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss (hunk 2 above)
Comment (AmsterGet) [reply app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss:110]: Usually the components from Design system are not subject to override, especially if the Design systems developed by us.
So please update colors inside of the `Button` component from `componentLibrary`.

Comment (BlazarQSO) [reply app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss]: Done.

File: app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.scss [hunk 3]
```diff
@@ -51,10 +51,22 @@
...
+
+    &:hover {
+      color: $COLOR--topaz-hover-2;
+    }
+    &:focus {
+      color: $COLOR--topaz-focused;
+    }
+    &:active {
+      color: $COLOR--topaz-pressed;
+    }
```
Comment (AmsterGet) [inline app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.scss:64]: Same here.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.scss (hunk 3 above)
Comment (BlazarQSO) [reply app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.scss]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**